src/util/string_container.cpp

```diff
@@ -87,3 +85,10 @@ unsigned string_containert::get(const std::string &s)
 
   return r;
 }
+
+/// Get a reference to the global string container.
+string_containert &get_string_container()
+{
+  static string_containert ret{};
```

Comment:
Could be done if done consistently across the code base. Else please remove them for we don't (yet) do it elsewhere.